tools/buildHelper/buildHelper.psm1

```diff
@@ -775,6 +758,68 @@ function Get-TagData
     }
 }
 
+function Invoke-FormatTagTemplate {
+    param(
+        [parameter(Mandatory)]
+        [string]
+        $TagTemplate,
+
+        [parameter(Mandatory)]
+        [object]
```

Comment:
if these are the result of `group-object` you might use `[Microsoft.PowerShell.Commands.GroupInfo[]]` rather than object.